ttt/src/main.rs

```diff
@@ -128,6 +129,18 @@ fn sync(ws: &WalletConn) -> Result<()> {
     Ok(())
 }
 
+fn history_log(index: usize) -> Result<()> {
```

Comment:
better use option instead of usize, when None is passed in we are getting all the history